department-of-veterans-affairs/caseflow
# 8616 remove special issues for ama

Resolves #8616

### Description

for AMA appeals, remove special issues for attorneys and judges.

This required changing route definitions, changing the way we pushed on to the browser history in the ActionsDropDown component, as well as changing back button urls which referenced special issues.  Redux was used to store the specialIssuesRedirect as well as whether an appeal was legacy or not, this was stored in `state.queue.appealDetails[appealId].isLegacy` and `state.queue.appealDetails[appealId].specialIssuesRedirect`.

### Acceptance Criteria

### Testing Plan
1. Login as an attorney or judge
2. go to queue
3a. select a legacy appeal -> select ready for decision -> you should see special issues -> then when you click next you should see dispositions
3b. select a non legacy appeal -> select ready for decision -> you should NOT see special issues and see dispositions instead. when you click back, you should go back to Case Details View
